.github/workflows/github-pages.yml

```diff
@@ -37,6 +37,9 @@ jobs:
           profile: minimal
           override: true
 
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
```

Comment:
Should we just be running these tests via the dev docker image instead? 🤔

Reply:
this is the pages site generation so it requires a bit more work to generate the pages in docker...can be done but this is probably easier for now.